include/Physics/Car.h

```diff
@@ -0,0 +1,41 @@
+//
+// Created by SpiritStudio on 28.11.18.
+//
+
+#ifndef VEHICLESEVOLUTION_CAR_H
+#define VEHICLESEVOLUTION_CAR_H
+
+#include <Box2D/Box2D.h>
+
+#include <Physics/CarBody.h>
+#include <Physics/Wheel.h>
+#include <EvolutionaryAlgorithm/CarParameters.h>
+
+class Car {
+
+public:
+    Car(b2World &world, const b2Vec2 &position, const CarParameters &car_parameters);
+
+    b2Vec2 getPosition() const;
+    b2Vec2 getFrontWheelPosition() const;
+    b2Vec2 getRearWheelPosition() const;
+    double getFrontWheelRadius() const;
+    double getRearWheelRadius() const;
+    std::vector<b2Vec2> getCarBodyVertices() const;
```

Comment:
There is an issue. You cannot return `const std::vector<b2Vec2>&` as it would be a reference to local object (check `CarBody::getPolygon()`). We need to add attribute to `CarBody` or leave as it is.